mqtt/mqtt-broker/tests/common.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(dead_code)]
```

Comment:
why dead code now?

Reply:
Some methods are not used in PacketStream. clippy shows warnings